data_generation/utils/secret_manager_utils.py

```diff
@@ -0,0 +1,14 @@
+import ast
+from google.cloud import secretmanager
+
+
+def get_secret_value_Dict(project_id="", secret_id="", version_id="latest"):
+
+    client = secretmanager.SecretManagerServiceClient()
+
+    secret_name = f"projects/{project_id}/secrets/{secret_id}/versions/{version_id}"
+
+    response = client.access_secret_version(request={"name": secret_name})
+    secret_value = ast.literal_eval(response.payload.data.decode("UTF-8"))
```

Comment:
why are we evaluating a string instead of just storing the string? obfuscation is not a good practice

Reply:
ast.literal_eval is converting this string (`response.payload.data.decode("UTF-8")`) into a dictionary. So the goal here is not obfuscation.